includes/polling/ports.inc.php

```diff
@@ -747,6 +747,10 @@
 
         influx_update($device, 'ports', rrd_array_filter($tags), $fields);
         graphite_update($device, 'ports|' . $ifName, $tags, $fields);
+           foreach ($fields as $k => $v) {
+                opentsdb_update($device, "port." . $k, array('ifName' => $this_port['ifName'],'ifIndex' =>  getPortRrdName($port_id)), array('key' => $v));
```

Comment:
Any reason this is here and not just done in `opentsdb_update()` and each metric has to be sent separately?